### PR TITLE
WIP: MSI - Onboard RHODS interop AWS IPI jobs

### PR DESCRIPTION
**Pull request by omrirh**

### CODE REVIEW

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Review by oharan2 — APPROVED** [review]

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

File: ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-production.yaml [hunk 1]
```diff
@@ -89,6 +89,36 @@ tests:
     test:
     - ref: addon-tests-rhods
     workflow: addons-and-operators
+- as: smoke-ipi-rhods-addon-baseline-fips
+  cron: 0 12 * * 6
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      ADDON1_CONFIG: name=managed-odh;notification-email=interop-qe-ms@redhat.com
+      CLUSTER1_CONFIG: name=rhods-aws-ipi;base_domain=aws.interop.ccitredhat.com;platform=aws;region=us-east-2;version=4.13;compute-machine-type=g4dn.2xlarge;timeout=90m;replicas=2;fips=true;ocm-env=production
```
Comment (rnetser) [inline ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-production.yaml:98]: please use cluster prefix and not a fix name

File: ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-production.yaml [hunk 2]
```diff
@@ -89,6 +89,36 @@ tests:
...
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "operator-install", "failure_type": "all", "classification": "Provisioning - Operators", "jira_project": "MSINTEROP", "jira_epic": "MSINTEROP-437"},
+          {"step": "operator-uninstall", "failure_type": "all", "classification": "Deprovisioning - Operators", "jira_project": "MSINTEROP", "jira_epic": "MSINTEROP-437"},
+          {"step": "addon-install", "failure_type": "all", "classification": "Provisioning - Addons", "jira_project": "MSINTEROP", "jira_epic": "MSINTEROP-437"},
+          {"step": "addon-uninstall", "failure_type": "all", "classification": "Deprovisioning - Addons", "jira_project": "MSINTEROP", "jira_epic": "MSINTEROP-437"},
+          {"step": "addon-tests-*", "failure_type": "all", "classification": "Addons tests", "jira_project": "RHODS", "jira_epic": "RHODS-10714"},
+          {"step": "cluster-*", "failure_type": "all", "classification":  "Cluster Creation - IPI", "jira_project": "MSINTEROP", "jira_epic": "MSINTEROP-437"},
```
Comment (rnetser) [inline ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-production.yaml:106]: please fix based on new changes

File: ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-production.yaml [hunk 3]
```diff
@@ -89,6 +89,36 @@ tests:
...
+        ]
+      FIREWATCH_DEFAULT_JIRA_PROJECT: MSINTEROP
+      FIREWATCH_JIRA_API_TOKEN_PATH: /tmp/secrets/jira/access_token_msi
+      OPERATOR1_CONFIG: name=gpu-operator-certified;namespace=nvidia-gpu-operator;source=certified-operators;target-namespaces=nvidia-gpu-operator,
+      OPERATOR2_CONFIG: name=servicemeshoperator
+      OPERATOR3_CONFIG: name=serverless-operator;namespace=openshift-serverless
+      OPERATOR4_CONFIG: name=openshift-pipelines-operator-rh;namespace=openshift-pipelines;channel=latest
+      OPERATOR5_CONFIG: name=nfd;namespace=openshift-nfd;target-namespaces=openshift-nfd,
+      PARALLEL: "false"
```
Comment (rnetser) [inline ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-production.yaml:115]: see change in https://github.com/openshift/release/pull/43360

File: ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-production.yaml (hunk 1 above)
Comment (omrirh) [reply ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-production.yaml]: done

File: ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-production.yaml (hunk 2 above)
Comment (omrirh) [reply ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-production.yaml:110]: done

File: ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-production.yaml (hunk 3 above)
Comment (omrirh) [reply ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-production.yaml]: done

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Comment by omrirh** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-ipi-rhods-addon-baseline-fips

**Pull request closed without merging**